app/views/movie/create.scala.html

```diff
@@ -0,0 +1,42 @@
+@import helper._
+@import modules.utility.twirl.CustomHelpers._
+@import modules.movie.{CreateMovieForm}
+@import modules.util.{Country, Language}
+@(movieForm: Form[CreateMovieForm], id: Option[Long] = None)(implicit request: MessagesRequestHeader, flash: Flash)
+
+@main("Create movie") {
```

Comment:
Since the form serves both creating and updating, page title should be conditional too.